docs/api.md

```diff
@@ -53,8 +53,10 @@
 ```{eval-rst}
 .. autosummary::
     :toctree: _autosummary
-
+    
+    ValidateEachChunk
```

Comment:
can you keep this in a separate section, to clearly delineate transforms vs function?

Reply:
actually this is probably fine for now

Reply:
Updated, anyway.